### PR TITLE
fixed Flow error/warning

### PR DESCRIPTION
**Pull request by mschwartz**
Added some braces in if statements.  Seems clearer, easier to set breakpoints in debugger, etc.

Fixed onIdle and offIdle to be const.  Seems safer, and trivially less performant.

Added $FlowFixMe comment in the right place to stop the error message.

### CODE REVIEW

**Comment by mschwartz** [comment]
@grgur please review.

I have limited permissions on this repo - I can't request a review, for example.  Nor can I create branches without creating a fork.

**Comment by mschwartz** [comment]
@grgur

File: src/index.js [hunk 1]
```diff
@@ -39,25 +39,27 @@ const isBrowser = !!(
...
-let onIdle = fallbackOnIdle;
-
-let offIdle = fallbackOffIdle;
-
-if (isBrowser && typeof window.requestIdleCallback === 'function') {
-  // This is what we are here for. We are safe to use requestIdleCallback
-  onIdle = window.requestIdleCallback;
-  offIdle = window.cancelIdleCallback;
-}
+const onIdle =
```
Comment (grgur) [inline src/index.js]: @mschwartz here's an idea
```js
const isIdleCallbackSupported = isBrowser && typeof window.requestIdleCallback === 'function';
const onIdle = isIdleCallbackSupported ? window.requestIdleCallback : fallbackOnIdle;
const offIdle = isIdleCallbackSupported ? window.cancelIdleCallback : fallbackOffIdle;
```

Comment (mschwartz) [reply src/index.js]: Looks fine, made the change.

**Pull request merged**